docs/import-roster-from-lms.md

```diff
@@ -0,0 +1,78 @@
+
+### Creating a new course roster
+
+To import a new course roster from your Learning Management, select the classroom you would like to import into. Then proceed through the following steps:
+
+   <div class="d-flex flex-justify-around">
+     <img src="/images/help/lms/roster-import/import-roster.gif" class="border" style="width: 75%;">
+   </div>
+
+1. Navigate to classroom settings within GitHub Classroom
+
+2. Select "Roster Management" from the sidebar
+
+3. On the "Add students" page which pops up, click "Import from your Learning Management System"
+
+4. Select how you would like to identify the imported students by:
+
+    - **User ID**: The Student ID as stored in your institution's Learning Management System
+    - **Names**: Student full names
+    - **Emails**: Student emails
+
+    _Note:_ If you do not see the kind of identifier you want, you may have to enable GitHub Classroom to receive student name and email information per your Learning Management System's privacy settings. See [troubleshooting](#troubleshooting) for more advice.
+    {: class="note"}
+
+5. That's it! You should now have your roster imported into GitHub Classroom.
+
+    _Note:_ If you encounter any errors, check to ensure you have enabled the roster membership service from within your Learning Management System. See [troubleshooting](#troubleshooting) for more advice.
+    {: class="note"}
+
+
+### Syncing an existing course roster
+
+Once you've created a course roster imported from you Learning Management System, keeping it up to date is easy!
+
+   <div class="d-flex flex-justify-around">
+     <img src="/images/help/lms/roster-import/sync-roster.gif" class="border" style="width: 75%;">
+   </div>
+
+1. Navigate to classroom settings within GitHub Classroom.
+
+2. Select "Roster Management" on the sidebar
+
+3. In the existing roster view, click "Sync from your Learning Management System" in
+   the top right corner
+
+4. Select how you would like to identify the imported students by:
+
+    - **User ID**: The Student ID as stored in your institution's Learning Management System
+    - **Names**: Student full names
+    - **Emails**: Student emails
+
+    _Note:_ If you do not see the kind of identifier you want, you may have to enable GitHub Classroom to receive student name and email information per your Learning Management System's privacy settings. See [troubleshooting](#troubleshooting) for more advice.
+    {: class="note"}
+
+5. That's it! Your roster should now be up to date with the latest course information from your Learning Management System.
+
+    _Note:_ If you encounter any errors, check to ensure you have enabled the roster membership service from within your Learning Management System. See [troubleshooting](#troubleshooting) for more advice.
+    {: class="note"}
+
+### Troubleshooting
+
+#### Unable to import students by Name or Email
+Lorem ipsum
```

Comment:
I'd recommend waiting to merge until these are written, or dropping the placeholder sections entirely (placeholder text in shipped docs is… embarrassing—I have stories 😳)

Reply:
😅 Thanks for noticing, I mainly put these here as a concrete reminder to actually get them done by release. I'll have them ready for the second round of review!